src/mysql_protocol.erl

```diff
@@ -71,21 +73,36 @@ handshake(Username, Password, Database, SockModule0, SSLOpts, Socket0,
             Error
     end.
 
-handshake_finish_or_switch_auth(Handshake = #handshake{status = Status}, Password,
+handshake_finish_or_switch_auth(Handshake = #handshake{status = Status, auth_plugin_name = PluginName}, Password,
```

Comment:
I'm prefer that we keep the line length to maximum 80. It is not strict, but this line can be split easily.